lib/gollum-lib/wiki.rb

```diff
@@ -579,6 +599,16 @@ def history_sanitizer
       @history_sanitizer ||= history_sanitization.to_sanitize
     end
 
+    def redirects
+      ::Gollum::Redirects.load(self)
+    end
+    
+    def add_redirect(old_path, new_path)
+      redirects = ::Gollum::Redirects.load(self)
```

Comment:
with the above `redirects` method, isn't this line superfluous?

Reply:
Fixed in new implementation.